### ISSUE TITLE
Create "Contact Us" section

### ISSUE THREAD

**Issue by simondorfman** [lavafinancial/landing#9]
## Acceptance Criteria

1. Create a new section at the bottom of the page with a header of "Contact Us"
2. Under the header, create a paragraph that is styled like this existing copy on the page "Our innovative solutions to unlock the true potential of our protocol" (e.g. `<p class="mt-5 text-xl text-lava-yellow">`)
    1. ...with this content:

Contact us by [joining our discord](https://discord.gg/lava-financial-913586044379750460), then scrolling down to the #🎫│support-tickets section and click the green button that says "Create Ticket".

**Comment by simondorfman** [comment]
@larry-lava & @Appljuze , 

How do you want this to work? A form that the user can submit? A link to an email address? Perhaps best would be to link to discord and show how to start a ticket?

**Comment by larry-lava** [comment]
Yeah I like the idea of linking to Discord and instructing the user how to open a ticket. The reality is that their issue will be addressed far quicker with this system than if we were to embed a contact form or link to an email

**Comment by simondorfman** [comment]
simon todo: write this up and get ready for scribe.
